tests/Tests.AzureAppConfiguration/FeatureManagementTests.cs

```diff
@@ -188,95 +420,85 @@ public class FeatureManagementTests
-                                    ""variant"": ""Small"",
-                                    ""users"": [
-                                        ""Alice"",
-                                        ""Bob""
-                                    ]
-                                }   
-		                    ],
-		                    ""group"": [
-			                    {
-				                    ""variant"": ""Big"",
-				                    ""groups"": [
-					                    ""Ring1""
-				                    ]
-			                    },
-                                {
-                                    ""variant"": ""Small"",
-                                    ""groups"": [
-                                        ""Ring2"",
-                                        ""Ring3""
-                                    ]
-                                }
-		                    ],
-		                    ""percentile"": [
-			                    {
-				                    ""variant"": ""Big"",
-				                    ""from"": 0,
-				                    ""to"": 50
-			                    },
-                                {
-                                    ""variant"": ""Small"",
-                                    ""from"": 50,
-                                    ""to"": 100
-                                }
-		                    ]
-	                    }
-                    }
-                    ",
-            label: default,
-            contentType: FeatureManagementConstants.ContentType + ";charset=utf-8",
-            eTag: new ETag("c3c231fd-39a0-4cb6-3237-4614474b92c1"));
+        List<ConfigurationSetting> _variantFeatureFlagCollection = new List<ConfigurationSetting>
+        {
+            ConfigurationModelFactory.ConfigurationSetting(
+                key: FeatureManagementConstants.FeatureFlagMarker + "VariantsFeature1",
+                value: @"
+                        {
+                            ""id"": ""VariantsFeature1"",
+                            ""enabled"": true,
+                            ""variants"": [
+		                    {
+			                    ""name"": ""Big"",
+			                    ""configuration_value"": ""600px""
+		                    },
+		                    {
+			                    ""name"": ""Small"",
+			                    ""configuration_reference"": ""ShoppingCart:Small"",
+			                    ""status_override"": ""Disabled""
+		                    }
+	                        ],
+	                        ""allocation"": {
+		                        ""seed"": ""13992821"",
+		                        ""default_when_disabled"": ""Small"",
+		                        ""default_when_enabled"": ""Small"",
+		                        ""user"": [
+			                        {
+				                        ""variant"": ""Big"",
+				                        ""users"": [
+					                        ""Marsha"",
+                                            ""John""
```

Comment:
Indentation inconsistent throughout this file. Might be worth running dotnet format on it.

Reply:
Tried to do this now, looks like it doesn't work the same with strings within code. I could try to fix it manually too, just not sure if there's a better way